data/ka'het/ka'het outfits.txt

```diff
@@ -223,3 +223,87 @@ outfit "Ka'het Reserve Accumulator"
 	"outfit space" -3
 	"energy capacity" 600
 	description "This tiny battery was developed for use by the Ka'mar drones as an emergency energy storage, in case their generators can't quite suffice for the high power demand of most Builder systems."
+
+# Engines:
+
+outfit "Maeri Engine Nacelles"
+	category "Engines"
+	"cost" 928000
+	thumbnail "outfit/ka'het maeri engine"
+	"mass" 52
+	"outfit space" -76
+	"engine capacity" -59
```

Comment:
52 mass, but 59 engine cap? The other engines here have a matching mass and engine capacity.

Reply:
fixed the mass